src/appleseed-max-impl/appleseedinteractive/interactivetilecallback.cpp

```diff
@@ -0,0 +1,53 @@
+
+// Interface header.
+#include "appleseedinteractive/interactivetilecallback.h"
+
+namespace asr = renderer;
+
+#define WM_TRIGGER_CALLBACK WM_USER + 4764
+
+InteractiveTileCallback::InteractiveTileCallback(
+    Bitmap*                         bitmap,
+    IIRenderMgr*                    iimanager,
+    asr::IRendererController*       render_controller
+    )
+    : TileCallback(bitmap, 0)
```

Comment:
Should I add dummy member variable?

Reply:
I'm surprised that it works at all with 0. Anyway, the good news is that it looks like we can remove completely that second argument (it's a pointer to an integer that represents the number of rendered tiles), because although the integer is incremented, its value is never used.

Regarding this question:

> Not sure if I can pass pointer to a member in a constructor

You totally can, but you have to be very careful about one thing: the _actual_ order of initialization is the order of _declaration_, not the order in which you write the initializations. This is important when you initialize one member with the value of another (that another member must be initialized before you use it).

Example:

    struct C
    {
        int m_a;
        int m_u;

        C()
          : m_u(1)
          , m_a(m_u) // problem: m_a is initialized first since it's declared first, but then m_u is still uninitialized!
        {
        }
}

Reply:
I've removed second argument from TileCallback.

Reply:
Ah, you're right. Instead of introducing a dummy variable, it would be cleaner to continue passing `nullptr` for this argument (to clearly signify that we don't use it) but to check that the pointer is not null before incrementing the pointed value.